usaspending_api/common/helpers.py

```diff
@@ -338,3 +338,11 @@ def timer(msg='', logging_func=print):
 
 CREATE INDEX ON awards(FY(field_name))
 """
+
+CORRECTED_CGAC_PG_FUNCTION_DEF = '''
```

Comment:
Should we do this for all FREC agencies?

Reply:
Are there any other violators that fall into this specific case where the FREC/CGAC needs to be corrected?

Reply:
Added additional agencies to be accounted for.